src/screens/EmpowerPlant.tsx

```diff
@@ -40,10 +40,15 @@ const EmpowerPlant = ({navigation}) => {
 
   const loadData = () => {
     setProductData(null);
+    let se, customerType, email;
+    Sentry.withScope(function(scope) {
+      [ se, customerType ] = [scope._tags.se, scope._tags.customerType ]
```

Comment:
@dgbailey what was the workaround for letting you take _tags data off of the scope? Wasn't it preventing you at first?

Reply:
@thinkocapo Turns out the compiler will warn about what is deemed inappropriate use of a protected field but do nothing to prevent it at runtime.